specification/communication/data-plane/Chat/stable/2023-11-07/communicationserviceschat.json

```diff
@@ -0,0 +1,1791 @@
+        "participantRemoved"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ChatMessageType",
+        "modelAsString": true
+      }
+    },
+    "SendChatMessageRequest": {
+      "description": "Details of the message to send.",
+      "required": [
+        "content"
+      ],
+      "type": "object",
+      "properties": {
+        "content": {
+          "description": "Chat message content.",
+          "type": "string",
+          "example": "Come one guys, lets go for lunch together."
+        },
+        "senderDisplayName": {
+          "description": "The display name of the chat message sender. This property is used to populate sender name for push notifications.",
+          "type": "string",
+          "example": "Bob Admin"
+        },
+        "type": {
+          "$ref": "#/definitions/ChatMessageType"
+        },
+        "metadata": {
+          "description": "Message metadata.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SendChatMessageResult": {
+      "description": "Result of the send message operation.",
+      "required": [
+        "id"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "A server-generated message id.",
+          "type": "string",
+          "example": "123456789"
+        }
+      }
+    },
+    "ChatParticipant": {
+      "description": "A participant of the chat thread.",
+      "required": [
+        "communicationIdentifier"
+      ],
+      "type": "object",
+      "properties": {
+        "communicationIdentifier": {
+          "$ref": "#/definitions/CommunicationIdentifierModel"
+        },
+        "displayName": {
+          "description": "Display name for the chat participant.",
+          "type": "string",
+          "example": "Bob"
+        },
+        "shareHistoryTime": {
+          "format": "date-time",
+          "description": "Time from which the chat history is shared with the participant. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2020-10-30T10:50:50Z"
+        }
+      }
+    },
+    "ChatAttachmentType": {
+      "description": "The type of attachment.",
+      "enum": [
+        "image"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ChatAttachmentType",
+        "modelAsString": true
+      }
+    },
+    "ChatAttachment": {
+      "description": "An attachment in a chat message.",
+      "required": [
+        "id",
+        "attachmentType"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Id of the attachment",
+          "type": "string",
+          "example": "A27C7079-626F-4D23-8BEE-582F2C376B5A"
+        },
+        "attachmentType": {
+          "$ref": "#/definitions/ChatAttachmentType"
+        },
+        "extension": {
```

Comment:
@mikekistler, @JeffreyRichter, we are getting some feedback on naming of this property. I was wondering if "fileType" makes sense?

an example would be:

```
"attachments": [
                    {
                        "id": "efecf033e5198a9c15c92a3163dc90e5",
                        "attachmentType": "image",
                        "fileType": "png",
                        "name": "img",
                        "url": "https://contoso.azurewebsites.net/chat/threads/19:efecf033e5198a9c15c92a3163dc90e5@thread.v2/images/0efecf033e5198a9c15c92a3163dc90e5/views/original?api-version=2023-11-07",
                        "previewUrl": ""https://contoso.azurewebsites.net/chat/threads/19:efecf033e5198a9c15c92a3163dc90e5@thread.v2/images/0efecf033e5198a9c15c92a3163dc90e5/views/small?api-version=2023-11-07"
                    },
                   {
                        "id": "efecf033e5198a9c15c92a3163dc90e5",
                        "attachmentType": "file",
                        "fileType": "pdf",
                        "name": "report.pdf",
 
                        "previewUrl": "[https://contoso.sharepoint.com/:personal/contoso_inc/efecf033e5198a9c15c92a3163dc90e5](https://contoso.sharepoint.com/:personal/contoso_inc/efecf033e5198a9c15c92a3163dc90e5?xsdata=MDV8MDF8fDgzZThkZTIxZjM1ODQxNzlmOTUzMDhkYmUyMzlmMGM3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgzNTI1MTA2ODM3MDc3MzF8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMM1JsWVcxekx6RTVPbVkyWkRVeVlXTTJORFkxWXpRd1pXRTRNR1JqT0RaaU9HSmxNemd5TldGaFFIUm9jbVZoWkM1emEzbHdaUzlqYUdGdWJtVnNjeTh4T1RwbU5tUTFNbUZqTmpRMk5XTTBNR1ZoT0RCa1l6ZzJZamhpWlRNNE1qVmhZVUIwYUhKbFlXUXVjMnQ1Y0dVdmJXVnpjMkZuWlhNdk1UWTVPVFkxTkRJMk5qZ3lOUT09fGY5Njg2Y2JlNGJhZjQ0YmRmOTUzMDhkYmUyMzlmMGM3fDg4NjE4ZjkzNzZkYTQzMjhhNDMzOThmY2E0MWQ4MTgw&sdata=R0Q1Q0pLcXFaRzJYbjQ1bWNOZTVGMXZ6R2U0WWIzN2d1UnJ1aE9wL0pVZz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cjopeng%40microsoft.com)["](https://global.chat.ppe.communication.microsoft.com/chat/threads/19:meeting_ZDk0ODlhNTgtMzAxNS00MzE5LTg2MGYtYzM4MGJkZmM5Njk5@thread.v2/messages/1677921907914/teamsInterop/images/0-canarycus-d3-efecf033e5198a9c15c92a3163dc90e5/views/small?api-version=2023-04-01-preview%22)
                    },
                   {
                        "id": "efecf033e5198a9c15c92a3163dc90e5",
                        "attachmentType": "file",
 
                        "name": "file_with_no_extension",
                        "previewUrl": "[https://contoso.sharepoint.com/:personal/contoso_inc/efecf033e5198a9c15c92a3163dc90e5](https://contoso.sharepoint.com/:personal/contoso_inc/efecf033e5198a9c15c92a3163dc90e5?xsdata=MDV8MDF8fDgzZThkZTIxZjM1ODQxNzlmOTUzMDhkYmUyMzlmMGM3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgzNTI1MTA2ODM3MDc3MzF8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMM1JsWVcxekx6RTVPbVkyWkRVeVlXTTJORFkxWXpRd1pXRTRNR1JqT0RaaU9HSmxNemd5TldGaFFIUm9jbVZoWkM1emEzbHdaUzlqYUdGdWJtVnNjeTh4T1RwbU5tUTFNbUZqTmpRMk5XTTBNR1ZoT0RCa1l6ZzJZamhpWlRNNE1qVmhZVUIwYUhKbFlXUXVjMnQ1Y0dVdmJXVnpjMkZuWlhNdk1UWTVPVFkxTkRJMk5qZ3lOUT09fGY5Njg2Y2JlNGJhZjQ0YmRmOTUzMDhkYmUyMzlmMGM3fDg4NjE4ZjkzNzZkYTQzMjhhNDMzOThmY2E0MWQ4MTgw&sdata=R0Q1Q0pLcXFaRzJYbjQ1bWNOZTVGMXZ6R2U0WWIzN2d1UnJ1aE9wL0pVZz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cjopeng%40microsoft.com)["](https://global.chat.ppe.communication.microsoft.com/chat/threads/19:meeting_ZDk0ODlhNTgtMzAxNS00MzE5LTg2MGYtYzM4MGJkZmM5Njk5@thread.v2/messages/1677921907914/teamsInterop/images/0-canarycus-d3-efecf033e5198a9c15c92a3163dc90e5/views/small?api-version=2023-04-01-preview%22)
                    }
]
```

Reply:
I'm fine with fileType